shared.go

```diff
@@ -101,7 +101,7 @@ type Tag struct {
 // Image is used in a few places.
 type Image struct {
 	CoverType string `json:"coverType"`
-	URL       string `json:"url"`
+	URL       string `json:"url,omitempty"`
```

Comment:
Every image has a URL as far as I know. What does this change fix?

Reply:
In requests, it seems that RemoteURL is required while URL isn't, but I'll doublecheck this one as well.

Reply:
oh I see. yeah some apps use URL some use RemoteURL. this is probably fine.